limits.go

```diff
@@ -55,6 +56,28 @@ func (l *LimitError) Error() string {
 	return ""
 }
 
+// MarshalJSON returns the LimitError in JSON format.
+func (l *LimitError) MarshalJSON() ([]byte, error) {
+	var typeString string
+	switch l.Type {
+	case SchemaTypeMap:
+		typeString = "map"
+	case SchemaTypeList:
+		typeString = "list"
```

Comment:
Will this make sense to users?  OpenAPIv3 has "array" and "object" types  not "map" and "list" ("object with additionalProperties" is how what most developers consider a "map" are represented in OpenAPIv3).

Reply:
I was wondering how weird it might be to receive a "missing limit" error on an object/how best to phrase it - I've switched the wording/constants to use "array" and "object."